web/react/components/navbar_dropdown.jsx

```diff
@@ -142,10 +142,10 @@ export default class NavbarDropdown extends React.Component {
             >
             </li>
         );
-        if (this.state.teams.length > 1 && this.state.currentTeam) {
```

Comment:
Removing this.state.currentTeam here causes the line `, currentTeam: TeamStore.getCurrent()` in the constructor to be irrelevant. Should be removed.